examples/node.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import logging
+import asyncio
+
+from kademlia.network import Server
+
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+handler.setFormatter(formatter)
+log = logging.getLogger('kademlia')
+log.addHandler(handler)
+log.setLevel(logging.DEBUG)
+
+server = Server()
+
+
+def parse_arguments():
+    # Create argument parser
+    parser = argparse.ArgumentParser()
+
+    # Optional arguments
+    parser.add_argument("-i", "--ip", help="IP address of node", type=str, default=None)
+    parser.add_argument("-p", "--port", help="port number of node", type=int, default=None)
+    parser.add_argument("-k", "--key", help="key", type=str, default=None)
+    parser.add_argument("-v", "--value", help="value", type=str, default=None)
```

Comment:
If the idea is to just have a generalized node script that could either be a bootstrap or just connect to other nodes - I don't think it would/should set a key/value (that would be up to set.py as an example)

Reply:
@bmuller Good point I'll remove these arguments including the `server.set` and `server.stop` functions